### ISSUE TITLE
Location always return null

### ISSUE THREAD

**Issue by XandrMaster** [philipplackner/WeatherApp#6]
Location always return null : [getCurrentLocation()](https://github.com/philipplackner/WeatherApp/blob/master/app/src/main/java/com/plcoding/weatherapp/data/location/DefaultLocationTracker.kt#L23) if user just start device and launch app first .
Workaround : launch any app which request first location data properly first then launch WeatherApp.
So uncompleted location logic ;)

**Comment by abhayy08** [comment]
Yeah, Same here I've tried making the app again and still it was the same, I'm a bit relaxed that I'm not the only one with this issue
